docs/winrt-apis-for-desktop.md

```diff
@@ -0,0 +1,59 @@
+# WinRT API changes for Desktop apps
+
+In WinUI 3 Preview 4, we have made changes to the way Windowing works in Desktop apps, by no longer creating a CoreWindow or CoreApplicationView. As a result, many WinRT APIs will no longer be supported in Dekstop apps. This document will list out APIs that have been affected by this change, with replacement or workaround APIs that will allow you to achieve the same functionality. 
+
+This doc will continue to be updated as our community files bugs and as we identify more workarounds/replacements. If you are encountering issues with an API not listed here, please go ahead and [file a bug](https://github.com/microsoft/microsoft-ui-xaml/issues/new?assignees=&labels=&template=bug_report.md&title=) on this repo with the API in question and what you are trying to achieve by using it. 
+
+## ApplicationView
+This API is no longer supported in Desktop apps, and neither are any of its members or other APIs that depend on it. Below is a list of workarounds the team has identified for members or dependencies of the ApplicationView class.
```

Comment:
I think we can drop this paragraph from each of the sections, but covering it at the top?

Reply:
Done